UnityProject/Assets/Scripts/Items/Food/Donut.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections;
+using UnityEngine;
+
+
+public class Donut : Edible
+{
+    [Tooltip("The amount of brute damage healed for a player who has a secuirty role.")]
+    public int SecurityHealBruteDamage = 5;
```

Comment:
```suggestion
    [SerialiezedField]
    private int securityHealBruteDamage = 5;
```

We don't really need to know this number anywhere else. Make it private and add the ``SerializedField`` attribute.